frontend/src/features/help/containers/HelpModalContentContainer.tsx

```diff
@@ -18,6 +23,10 @@ interface IHelpModalContentContainerProps {
 const getHelpPageByLocation = (location: any): IHelpPage | undefined => {
   let helpPage: IHelpPage | undefined = undefined;
   helpPages.forEach((page, path) => {
+    /** need to check paramaters here outside of the pathname as mapview and parcel detail form now share same pathname */
+    if (location.search.includes(HelpPageKeys.PROPERTY_DETAIL_PAGE)) {
+      helpPage = PropertyDetailViewHelpPage;
```

Comment:
Have to check based off the params in order to see the sidebar. Which means we cannot use `location.pathname` but `location.search` for this scenario